next/test/fields.test.ts

```diff
@@ -278,6 +278,7 @@ describe('fields', () => {
           options: [
             { label: 'Active', value: 'active' },
             { label: 'Inactive', value: 'inactive' },
+            { label: 'Undefined', value: undefined },
```

Comment:
hi! this is correct! In JSON Schemas, a `oneOf` can have more than `const`, for example it can be `pattern` too. :)